libs/application/templates/new-primary-school/src/forms/Prerequisites.ts

```diff
@@ -82,6 +84,25 @@ export const Prerequisites: Form = buildForm({
             }),
           ],
         }),
+        buildMultiField({
+          id: 'hasNoChildren',
+          title: newPrimarySchoolMessages.pre.noChildrenFoundTitle,
+          description: newPrimarySchoolMessages.pre.noChildrenFoundDescription,
+          children: [
+            buildDescriptionField({
+              id: 'noChildrenDecsription',
+              title: '',
+              description: newPrimarySchoolMessages.pre.noChildrenFoundReasons,
+            }),
+
+            // Empty submit field to hide all buttons in the footer
+            buildSubmitField({
+              id: '',
+              title: '',
+              actions: [],
+            }),
```

Comment:
Is this still needed if you have `renderLastScreenButton` and `renderLastScreenBackButton`?
```suggestion
```

Reply:
Fixed